fmc/README.md

```diff
@@ -167,63 +182,103 @@ This field provides the Handle of the DV entry that stores the physical address
 
 ### fmc_tci_dv_hdl
 
-This field provides the Handle into the Data Vault where the TCI<sub>FMC</sub> is stored. TCI<sub>FMC</sub> is a SHA-384 Hash of the FMC Module.
+This field provides the Handle into the DCCM datavault where the TCI<sub>FMC</sub> is stored. TCI<sub>FMC</sub> is a SHA-384 Hash of the FMC Module.
 
-### fmc_cdi_kv_hdl
+### fmc_cdi_ecdsa_kv_hdl
 
-This field provides the Handle into the Key Vault where the CDI<sub>FMC</sub> is stored.
+This field provides the Handle into the Key Vault where the ECDSA CDI<sub>FMC</sub> is stored.
 
-### fmc_priv_key_kv_hdl
+### fmc_cdi_mldsa_kv_hdl
```

Comment:
Same comment from the other PRs in this series - the CDI is algorithm-agnostic, and we can just have one.

Reply:
On the previous ROM PR, we discussed not having the reconditioned UDS and FE for MLDSA key generation and to use the different strings (e.g. "idevid_cdi" for ECDSA and "idevid_mldsa_cdi") for deriving ECC and MLDSA CDIs. Are we now saying even that is not needed and to use different strings (e.g. "idevid_keygen" and "idevid_mldsa_keygen") for generating different seeds for generating the ECC and MLDSA keys but keeping the CDI the same?

Reply:
Ah yes that is what I was meaning to get across earlier. I was not meaning to use different strings to fork the CDI tree. I mean that we can keep the same CDI tree and different KDF labels when making asymmetric keys.

Reply:
Updated to use the same CDI.